site/content/zh/_index.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Envoy Gateway
+---
+
+{{< blocks/cover title="欢迎访问 Envoy Gateway！" image_anchor="top" height="full" >}}
+<a class="btn btn-lg btn-primary me-3 mb-4" href="/v0.6.0">
+  开始使用 <i class="fas fa-arrow-alt-circle-right ms-2"></i>
+</a>
+<a class="btn btn-lg btn-secondary me-3 mb-4" href="/v0.6.0/contributions">
+  参与贡献 <i class="fa fa-heartbeat ms-2 "></i>
+</a>
+<p class="lead mt-5">将 Envoy 代理作为独立或基于 Kubernetes 的应用程序网关进行管理</p>
```

Comment:
IMO, EG's capabilities make it better fit for the term "Application Gateway" rather than "API Gateway". However, let's keep it consistent with the English **version.
**
```suggestion
<p class="lead mt-5">将 Envoy 代理作为独立或基于 Kubernetes 的 API 网关进行管理</p>
```